src/configManager/models/database/sequelize/sequelizeConnector.js

```diff
@@ -65,7 +65,7 @@ async function initSchemas() {
             primaryKey: true
         },
         value: {
-            type: Sequelize.DataTypes.STRING
+            type: Sequelize.DataTypes.STRING(350)
```

Comment:
a little bit problematic
because if someone already using it with predator we will need to do migration for sequlize
otherwise the string length won't change